features/matchers/have_http_status_matcher.feature

```diff
@@ -136,7 +162,6 @@ Feature: `have_http_status` matcher
           visit "/widgets/new"
           expect(page).to have_http_status(200)
 
-          fill_in "Name", :with => "My Widget"
```

Comment:
There's no issue here. In rails 5.1 they've changed the form generation logic so this finder fails. However, this `fill_in` has nothing to do with the test. I think we should remove it 

1. for compatibility
2. because it doesn't aid in being documenting of this feature